riak_repl/tests/test_check.py

```diff
@@ -75,11 +74,7 @@ def test_service_check(aggregator, riak_server):
 
     c = RiakReplCheck('riak_repl', init_config, {}, None)
 
-    instance = {
-        'default_timeout': 5,
-        'url': 'http://{}:8098/riak-repl/stats'.format(get_docker_hostname())
-    }
-    c.check(instance)
+    c.check(dd_environment)
```

Comment:
Can you make the configuration a constant and use it here instead of the return value of the fixture directly

Reply:
Can you explain a bit why? I thought one point of the fixture was to return the value that was going to be used for the check. Otherwise the return from the fixture is useless, no?

Reply:
This specific fixture is a bit different. The returned value is used by `ddev` when spinning up the environment (the config yielded is then written to a file used as the configuration for the agent).

I wanted that mainly to do like we did in every other check. Also usually in the tests we use different instances to test for different things, that's why we don't use directly the return value of the fixture.

I guess for this one it works though. I don't have a strong opinion.

Reply:
That makes sense, let me change it.